File: test/day1_test.exs
defmodule Aoc.Year2017.Day1Test do
    use ExUnit.Case
  
    test "PART 1 -- should make the sum if the next digit is the same" do
      assert Aoc.Year2017.Day1.part1("1122") == 3
      assert Aoc.Year2017.Day1.part1("1111") == 4
      assert Aoc.Year2017.Day1.part1("1234") == 0
      assert Aoc.Year2017.Day1.part1("91212129") == 9
    end
    
    # test "PART 2 -- should make the sum if the digit around are the same" do
    #   assert Aoc.Year2017.Day1.part2("1212") == 6
    #   assert Aoc.Year2017.Day1.part2("1221") == 0
    #   assert Aoc.Year2017.Day1.part2("123425") == 4
    #   assert Aoc.Year2017.Day1.part2("123123") == 12
    #   assert Aoc.Year2017.Day1.part2("12131415") == 4
    # end
  end
  

File: day1/day1.test.js
const solution = require("./index");

describe("2017 -- DAY 1", () => {
  it("should make the sum if the next digit is the same", () => {
    expect(solution("1122")).toBe(3);
    expect(solution("1111")).toBe(4);
    expect(solution("1234")).toBe(0);
    expect(solution("91212129")).toBe(9);
  });
});
